File: Python Word examples/paragraphs/getParagraphFormat.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.paragraphs_api.ParagraphsApi(configuration)

file_name = "Template.docx"
node_path = "sections/0"
index = 1
result = api.get_paragraph_format(file_name, node_path, index, folder="input")

File: JavaScript Word Examples/ParagraphsApiDemo/getParagraphFormat.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.ParagraphsApi(apiClient);

    var filename = "GetParagraphFormat.docx";
    var nodePath = "Section/0/Body/0";
    var opt = {
        folder: 'input'
    };
    var index = 0;
    instance.getParagraphFormat(filename, nodePath, index, opt, function (error, data, response) {
        if (error) throw error;
    });
})();
